spec.html

```diff
@@ -6996,20 +7007,15 @@ <h1>
       </emu-note>
     </emu-clause>
 
-    <emu-clause id="sec-iterabletolist" type="abstract operation">
+    <emu-clause id="sec-iteratortolist" oldids="sec-iterabletolist" type="abstract operation">
       <h1>
-        IterableToList (
-          _items_: an ECMAScript language value,
-          optional _method_: a function object,
+        IteratorToList (
+          _iteratorRecord_: an Iterator Record
         ): either a normal completion containing a List of ECMAScript language values or a throw completion
```

Comment:
@ptomato Assuming this lands, [Temporal IterableToListOfType](https://tc39.es/proposal-temporal/#sec-iterabletolistoftype) should probably change accordingly.

Reply:
Thank you.